src/settings/MappingProfiles/detailsSections/edit/MappingMARCBibDetails.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MARCTable } from '../../../../components/MARCTable';
+import { OverrideProtectedFieldsTable } from './MARCDetailsSection';
+
+import {
+  FIELD_MAPPINGS_FOR_MARC,
+  marcFieldProtectionSettingsShape,
+} from '../../../../utils';
+
+export const MappingMARCBibDetails = ({
+  marcMappingDetails,
+  fieldMappingsForMARCField,
+  marcFieldProtectionFields,
+  mappingMarcFieldProtectionFields,
+  setReferenceTables,
+}) => {
+  const marcTableFields = marcMappingDetails ||
+    [{
+      order: 0,
+      field: { subfields: [{}] },
+    }];
+  const defaultFieldMappingForMARCColumns = ['arrows', 'action', 'field', 'indicator1', 'indicator2',
+    'subfield', 'subaction', 'data', 'position', 'addRemove'];
+
+  const renderUpdatesDetails = () => {
+    const updatesFieldMappingForMARCColumns = ['arrows', 'field', 'indicator1', 'indicator2', 'subfield', 'addRemove'];
+
+    return (
+      <>
+        <MARCTable
+          columns={updatesFieldMappingForMARCColumns}
+          fields={marcTableFields}
+          onChange={setReferenceTables}
+        />
+        <OverrideProtectedFieldsTable
+          marcFieldProtectionFields={marcFieldProtectionFields}
+          mappingMarcFieldProtectionFields={mappingMarcFieldProtectionFields}
+          setReferenceTables={setReferenceTables}
+        />
+      </>
+    );
+  };
+
+  const renderMappingMARCBibDetails = () => {
+    if (fieldMappingsForMARCField === FIELD_MAPPINGS_FOR_MARC.UPDATES) {
+      return renderUpdatesDetails();
+    }
+
+    return (
+      <MARCTable
+        columns={defaultFieldMappingForMARCColumns}
+        fields={marcTableFields}
+        onChange={setReferenceTables}
+      />
+    );
+  };
+
+  return fieldMappingsForMARCField && renderMappingMARCBibDetails();
+};
+
+MappingMARCBibDetails.propTypes = {
+  marcMappingDetails: PropTypes.arrayOf(PropTypes.object.isRequired),
+  marcFieldProtectionFields: PropTypes.arrayOf(marcFieldProtectionSettingsShape),
```

Comment:
Add default values for these props. For `marcMappingDetails` it can be an array from lines 20-23